tardis/transport/montecarlo/base.py

```diff
@@ -223,6 +224,10 @@ def run(
             self.montecarlo_configuration.ENABLE_VPACKET_TRACKING
         )
 
+        self.transport_state.rpacket_last_interaction_trackers = (
```

Comment:
Is there a reason these added lines are not under the if statement on line 214?

Reply:
I will change this in the later iterations of this PR.

Reply:
I have changed the code now. So, what I am doing now is, if RPacket Tracking is enabled, track all the interactions; otherwise, track only the last interactions. @DeerWhale

Reply:
I added a flag just for the last interactions.